py3status/modules/weather_owm.py

```diff
@@ -261,11 +261,12 @@
 """
 
 import datetime
+import json
 
 # API information
 OWM_CURR_ENDPOINT = "https://api.openweathermap.org/data/2.5/weather?"
 OWM_FUTURE_ENDPOINT = "https://api.openweathermap.org/data/2.5/forecast?"
-OWM_ONECALL_ENDPOINT = "https://api.openweathermap.org/data/2.5/onecall?"
```

Comment:
(Thought) We might be able to use onecall 3.0 here (it contains 4 endpoints) for everything. There is a `current` and `daily` data that may be able to replace this. Deep module changes because of new API. Might as well do it all the way due to new API keys and billing subscription plan.

Reply:
I'll volunteer for any testing. I don't come close to the 1000/day; and, I've limited my total calls/day in my plan as well.

Reply:
We'll cover that on a separate PR. For now we'll focus on fixing the current implementation with a salt of safety.